.github/workflows/pytest.yml

```diff
@@ -198,37 +221,40 @@ jobs:
         with:
           path: coverage
```

Comment:
Why do we use `path: coverage` here but when we're uploading the artifact above we use `path: coverage.xml`?